.github/workflows/main.yml

```diff
@@ -0,0 +1,60 @@
+name: main
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - 'main'
+    types: [opened, synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      # TODO: Change this step after appling spotless.
+      # - name: lint
+      #   run: ./gradlew checkstyle
+      - run: ./gradlew test
+
+      - name: Show Current Version
+        run: ./gradlew printVersion
+      - name: Build Gem & Put it into './build/gems/' Directory
+        run: ./gradlew gem
+
+      - name: Set up JRuby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: jruby
+
+      - name: Publish to GPR
+        if: startsWith( github.ref, 'refs/tags/' )
```

Comment:
This step runs only when tagging into this repository.

Reply:
So, I have not been able to check the step works correctly. I will check when tagging the next.